gvm/connections.py

```diff
@@ -97,7 +97,7 @@ class GvmConnection(XmlReader):
 
     def __init__(self, timeout: Optional[int] = DEFAULT_TIMEOUT):
         self._socket = None
-        self._timeout = timeout if timeout is not None else DEFAULT_TIMEOUT
```

Comment:
I am not sure if this is correct. A timeout of 0 has a special meaning and it wouldn't be possible to set it anymore.

Reply:
Did not considered this ...